City-of-Helsinki/helsinki-paatokset
# PP-0: Update trustee migration to only include council members

Note: You need to be connected to the VPN for this to work. If you get empty results from any of the commands, there might be a connection error with the API, and the rest of the steps will not work.

**To test**
- Checkout branch, SSH into the container and run `drush cr`
- Make sure the Open ID connection is configured and the access token is set: https://helsinki-paatokset.docker.so/fi/admin/ahjo-open-id/
- Run `drush ap:pt -v`, this should fetch all positions of trust for all organizations and council groups and should update the positionsoftrust.json file.
- Run `drush ap:trust -v`, this should fetch all individual people for all organizations and council groups, and update the trustees.json file
- Run `drush ap:cpt -v`, this should fetch positions of trust only for the city council and government and update teh positionsoftrust_council.json file
- Run `drush ap:trust positionsoftrust_council.json -v`, this should fetch the trustees only for the council, and update the trustees_council.json file
- Run `drush migrate-status --tag=ahjo_api_trustees`. This should show migrations for ahjo_trustees:all, with about 300 items and ahjo_trustees:council, with about 150 items.
